github-site-plugin/src/main/java/com/github/maven/plugins/site/SiteMojo.java

```diff
@@ -433,7 +433,7 @@ public void execute() throws MojoExecutionException {
 
             CommitUser author = new CommitUser();
             author.setName(user.getName());
-            author.setEmail(user.getEmail());
+            author.setEmail(userService.getEmails().get(0));
```

Comment:
that might work as well; you'd have to give it a try.
I don't know the specifics of of the egit api anymore and don't have time atm.